Remotion/Web/Core/Res/Themes/NovaViso/Image/sprite.svg

```diff
@@ -106,7 +127,12 @@
     <use href="#TreeViewClosed" />
   </svg>
   <svg class="sprite-part" id="TreeViewWhite" width="16px" height="16px" viewBox="0 0 16 16" />
-  <image id="ValidationError" height="17" width="12" href="data:image/gif;base64,R0lGODlhDAARAPcAAAAAAP8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACH5BAEAAP8ALAAAAAAMABEAAAgzAP8JFBggwMCDBAsaRJiwIMOGCxkqjIhw4sN/Fi86vIiRokSPFUFyHMlx48OMH01+RBgQADs=" />
+  <svg class="sprite-part" id="ValidationError" width="16px" height="16px" viewBox="0 0 16 16">
+    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd" stroke-linecap="round">
```

Comment:
For the validation icon, we already have a definition in zeplin which looks quite like this one but it is more compact. could you please switch.

We should also switch to the darker red used in the font to match the colors.

lastly, the diameter of the disk is a bit smaller than the diameter of the help icon. could you please match the validation marker to the help icon. Perhaps we can/should just switch the disk path to an acutal filled circle. Would make things easier to read and size?

Reply:
> For the validation icon, we already have a definition in zeplin which looks quite like this one but it is more compact.

which one?

Reply:
ico/alert
![image](https://user-images.githubusercontent.com/6972548/160126548-53bd61ff-2c27-4ba8-87cf-a55e9e437d35.png)